lib/chef/resource/execute.rb

```diff
@@ -509,7 +509,6 @@ class Execute < Chef::Resource
       def initialize(name, run_context = nil)
         super
         @command = name
-        @backup = 5
```

Comment:
where did this default go?

Reply:
I bet this is leftover from when backup wasn't a property

Reply:
It's probably this: https://github.com/chef/chef/blob/master/lib/chef/resource/file.rb#L65-L66